src/lib/libcore.js

```diff
@@ -1627,6 +1631,11 @@ addToLibrary({
 
     for (var __exportedFunc in wasmExports) {
       var jsname = asmjsMangle(__exportedFunc);
+#if DYNCALLS || !WASM_BIGINT
+      if (jsname.startsWith('dynCall_')) {
+        dynCalls[jsname.substr(8)] = wasmExports[__exportedFunc];
```

Comment:
Return here, or `else`, we don't need these symbols globally.

Reply:
But of course we do? User JS code might call `dynCall_ii(func, i1, i2)` to call a JS pointer?

Reply:
Yeh, for regular users they'll still go out to the global scope.